core/src/main/scala/core/database/QueryRunner.scala

```diff
@@ -4,26 +4,36 @@ import javax.sql.DataSource
 
 import scala.concurrent.{ExecutionContext, Future}
 
-class QueryRunner(
-    db: Database,
-    ec: ExecutionContext
-) {
-  def run[A](query: Query[A]): Future[A] =
+/**
+  * A class who can run a Query.
+  */
+sealed trait QueryRunner {
+  val ds: DataSource
```

Comment:
Would not put such dependency on Play JDBC (`Database`) in core: if I want to use `Query` with a standalone Anorm app without Play, it's over ...